<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectionGuessDialog</class>
 <widget class="QDialog" name="ProjectionGuessDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vertex Projection Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Configure your application's projection matrix used for final vertex transform, which will be used to unproject rasterized vertices into world space.&lt;/p&gt;&lt;p&gt;Some parameters can be empty, which will use an estimate/guess based on the current mesh data.&lt;/p&gt;&lt;p&gt;For reverse-Z you can enable an infinite far plane.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_5">
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item row="2" column="1">
      <widget class="QLineEdit" name="aspectRatio">
       <property name="placeholderText">
        <string>Auto</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="nearPlane">
       <property name="placeholderText">
        <string>Auto</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="fov">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>179.000000000000000</double>
       </property>
       <property name="value">
        <double>90.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Far Plane:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Aspect Ratio:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="farPlane">
       <property name="placeholderText">
        <string>Auto</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Matrix type</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Near Plane:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>FOV:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="matrixType">
       <item>
        <property name="text">
         <string>Perspective</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Orthographic</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="infiniteFar">
       <property name="text">
        <string>Infinite</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>matrixType</tabstop>
  <tabstop>fov</tabstop>
  <tabstop>aspectRatio</tabstop>
  <tabstop>nearPlane</tabstop>
  <tabstop>farPlane</tabstop>
  <tabstop>infiniteFar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
